### ISSUE TITLE
Add example to config describing how to power off monitors after locking

### ISSUE THREAD

**Issue by Hofer-Julian** [YaLTeR/niri#856]
With the current default behavior, `Mod + Alt + L` opens `swaylock`, but doesn't power off monitors. The config should include a commented out example showing how to add the power-off after logging out.

**Comment by BRS5672023** [comment]
Actually I think this might be the desired behavior.. One can add a rule in swayidle so that if swaylock is running, then after some time the screen is turned off. For example, I'm using the following swayidle config:
```
timeout 900 'pidof swaylock || swaylock -f'
timeout 960 'niri msg action power-off-monitors'
timeout 60 'pidof swaylock && niri msg action power-off-monitors'
before-sleep 'pidof swaylock || swaylock -f'
```

**Comment by YaLTeR** [comment]
Oh, that's an interesting way to do it.

**Comment by wzbfyb** [comment]
I configured it like this, I guess this could be added to the example config?

`    Super+Alt+L hotkey-overlay-title="Lock the Screen: swaylock" { spawn-sh "niri msg action power-off-monitors ; swaylock";}
`

**Comment by tomasciccola** [comment]
Yeah, one can use niri's own ipc mechanism to turn monitors off. Buut, if niri supported `wlr-output-power-managment-v1`, one could do that in a compositor agnostic way

**Comment by wzbfyb** [comment]
sure, but I do not quite get why you need to do that in a compositor agnostic way while using a specific compositor.
anyhow I think this example is good enough for the documentation and saves people from googling around.

**Comment by tomasciccola** [comment]
because that way, you can have scripts that don't depend on a compositor. You can play/try other compositors and a lot of stuff will just keep working.
f.e. I have a script to focus an open window or launch an app (using fuzzel). Since I discover [wlrctl](https://git.sr.ht/~brocellous/wlrctl), I replace the part of the script for the window selection from using `niri msg` to using `wlrctl`. 
In the end its just preference, but I like not depending on compositors IPCs mechanisms
